config/config.go

```diff
@@ -136,6 +136,7 @@ func (cfg *Log) validateAndLog() {
 type LogLevel string
 
 const (
+	Trace   LogLevel = "trace"
```

Comment:
Since we no longer have a switch case to determine what level to log the error at, are these constants still needed?

Reply:
Looks like these constants are used as kind of an enum for setting up application log level on start up so makes sense to leave them.